operator/validator/controller.go

```diff
@@ -1080,45 +1061,13 @@ func (c *controller) setShareFeeRecipient(share *ssvtypes.SSVShare, getRecipient
 	return nil
 }
 
-func (c *controller) validatorStart(validator *validators.ValidatorContainer) (bool, error) {
-	if c.validatorStartFunc == nil {
-		return validator.Start(c.logger)
-	}
-	return c.validatorStartFunc(validator)
-}
-
-//func (c *controller) startValidatorAndCommittee(v *val)
-
 // startValidator will start the given validator if applicable
 func (c *controller) startValidator(v *validators.ValidatorContainer) (bool, error) {
 	c.reportValidatorStatus(v.Share().ValidatorPubKey[:], v.Share().BeaconMetadata)
 	if v.Share().BeaconMetadata.Index == 0 {
 		return false, errors.New("could not start validator: index not found")
 	}
-	started, err := c.validatorStart(v)
-	if err != nil {
-		c.metrics.ValidatorError(v.Share().ValidatorPubKey[:])
-		return false, errors.Wrap(err, "could not start validator")
-	}
-	if started {
-		c.recentlyStartedValidators++
-	}
-
-	return true, nil
```

Comment:
doesn't this mean we're not starting validators at all right now?

Reply:
I agree we should call Validator.Start here.